experiments/snippets/realTimeOppDashboard/README.md

```diff
@@ -0,0 +1,7 @@
+# Real Time Opportunity Dashboard
```

Comment:
As is, this page *does not work*. 

I did get it working by using an action sequence, but if we want this to be more universal, and since we're already in snippet land, we have two options:

- Change `poll for updates` into an In-Line (Snippet) snippet type. Create an event-triggered action sequence that then runs that snippet on Skuid Page: Rendered
- Wrapping the snippet with our recommended method of waiting for page rendering as of 11.1.0:

```
$(document.body).one('pageload',function() { 
// Will fire after Skuid is finished fully loading the page
// Code goes here
})
```

Reply:
I added the code to the snippet.  It seems to work now.